copier.yml

```diff
@@ -111,13 +112,13 @@ _tasks:
   # Initial commit
   - "{% if initial_commit %}git init{% endif %}"
   - "{% if initial_commit %}git add .{% endif %}"
-  - "{% if initial_commit %}git commit -m 'chore: initial commit'{% endif %}"
+  - '{% if initial_commit %}git commit -m "chore: initial commit"{% endif %}'
```

Comment:
Curious, why did you change the quotes style?

Reply:
Single quotes doesn't work on Windows

Reply:
Oh nice TIL - Thanks